history/kadai-simplehistory-provider/src/test/java/acceptance/events/workbasket/CreateHistoryEventOnWorkbasketMarkedForDeletionAccTest.java

```diff
@@ -60,7 +60,7 @@ void should_CreateWorkbasketMarkedForDeletionHistoryEvent_When_WorkbasketIsMarke
 
     String eventType = events.get(0).getEventType();
     String eventWorkbasketId = events.get(0).getWorkbasketId();
-    String details = workbasketHistoryEventMapper.findById(events.get(0).getId()).getDetails();
+    workbasketHistoryEventMapper.findById(events.get(0).getId());
```

Comment:
Why do we need this line? Could we check the value of "details" instead of deleting the variable?

Reply:
I don't want to change any logic or tests in this PR. I deleted the line altogether